dm/tests/check_task/run.sh

```diff
@@ -36,7 +30,10 @@ function test_check_task_fail_no_block() {
 }
 
 function test_check_task_fail_no_block_forsharding() {
-	prepare_many_tables
+	run_sql_both_source "drop database if exists \`check-task\`"
+	run_sql_both_source "create database if not exists \`check-task\`"
+	run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+
 	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"check-task $cur/conf/task-sharding.yaml" \
 		"\"state\": \"fail\"" 1
```

Comment:
cleanup_data some lines below can also clean `check-task`

rest lgtm

Reply:
fixed in https://github.com/pingcap/tiflow/commit/3c0284bcea3647d2681524e814439ef4b0694752

Reply:
I mean cleanup_data checktask check-task in line 50, 54

You change will not be executed if test fails before it and will not clean the environment before `run`

Reply:
https://github.com/pingcap/tiflow/pull/5896/commits/42a58a2a63f6ee50d297efe16babb2dc57f7cd0b 😬